AirCasting/CreateSessionViews/CreatingSessionFlowRootView.swift

```diff
@@ -8,19 +8,18 @@
 import SwiftUI
 
 struct CreatingSessionFlowRootView<Content: View>: View {
-    
     let content: () -> Content
-    
+
     @Environment(\.presentationMode) var presentationMode
-    
+
```

Comment:
please don't use cmd+a + ctrl+i :D this makes it hard to read thru CR and will be source of false conflicts